### ISSUE TITLE
Produces not authorized (401) status

### ISSUE THREAD

**Issue by davidcreager** [grdn1337/node-red-contrib-apple-findme-reloaded#3]
Think Apple have done something

**Comment by tb1337** [comment]
Yes. Currently figuring out what to do to get things working again.

**Comment by photoangell** [comment]
It would be useful to have the http status returned by the node in all cases. 

This way, any non http 200 status could be picked up by the rest of the flow and handled appropriately.

**Comment by tb1337** [comment]
Can do that, but slightly differenz: it will produce an error message with status code information, which you may catch, if you want. But before I continue developing this, I must check if it is possible to get this node working again.

**Comment by zyce78** [comment]
Any news from Apple if this will be available again?

**Comment by tb1337** [comment]
Seems like the access without 2FA doesn't work anymore, still investigating. Its not an official third-party API, we just utilized it cause we could.

**Comment by GeorgyBenelli** [comment]
> Seems like the access without 2FA doesn't work anymore, still investigating. Its not an official third-party API, we just utilized it cause we could.

seem to be some progress over at https://github.com/openhab/openhab-addons/issues/13282

**Comment by jjhall99** [comment]
with 2FA turned on, app specific passwords are available...
